### PR TITLE
Made the Testing Documents

### PR DESCRIPTION
**Pull request by Thomas-Frew**
- Talked about self-testing
- Talked about interoperability testing
- Removed Herobrine

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+![C++](https://img.shields.io/badge/c++-%2300599C.svg?style=for-the-badge&logo=c%2B%2B&logoColor=white)
+![Python](https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)
+
+
+We are proud to present the user/interoperability testing framework we used for our CLI chat app, based on [OLAF-Neighbourhood protocol](https://github.com/xvk-64/2024-secure-programming-protocol).
+
+This document contains our testing suite and approaches for user/iteroperability testing.
+
+## Test List
```
Comment (mortonjack) [inline TESTING.md:11]: Should this maybe be a "Requirements List"?

File: assets/Interoperability Cases.drawio.png:Zone.Identifier [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=https://app.diagrams.net/
```
Comment (mortonjack) [inline assets/Interoperability Cases.drawio.png:Zone.Identifier:3]: Why is this file in here

**Review by mortonjack — CHANGES_REQUESTED** [review]

File: assets/Interoperability Cases.drawio.png:Zone.Identifier (hunk 2 above)
Comment (Thomas-Frew) [reply assets/Interoperability Cases.drawio.png:Zone.Identifier:3]: :3

File: TESTING.md (hunk 1 above)
Comment (Thomas-Frew) [reply TESTING.md:11]: I think "Test List" captures the idea better, this is a testing document after all, and we are testing for them.

**Review by mortonjack — APPROVED** [review]

**Pull request merged**